TapCounter/ViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ViewController.swift
+//  TapCounter
+//
+//  Created by Alice Aponasko on 1/24/16.
+//  Copyright © 2016 justforfun. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
@AliceAponasko yep, sure. It's strange, because I've left inline comment. 
As for name, I guess both of your variant are good. Or you can name it just MainVC. Thanks.